cross/libpam/Makefile

```diff
@@ -0,0 +1,21 @@
+PKG_NAME = Linux-PAM
+PKG_VERS = 1.5.2
+PKG_EXT = tar.xz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/linux-pam/linux-pam/releases/download/v1.5.2
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+HOMEPAGE = https://github.com/linux-pam/linux-pam
+COMMENT = Linux PAM (Pluggable Authentication Modules for Linux) project
+LICENSE = Custom
```

Comment:
Isn't it the GNU General Public License?

Reply:
> ALTERNATIVELY, this product may be distributed under the terms of the
GNU General Public License, in which case the provisions of the GNU
GPL are required INSTEAD OF the above restrictions.  (This clause is
necessary due to a potential conflict between the GNU GPL and the
restrictions contained in a BSD-style copyright.)